runtime/server_http_request.go

```diff
@@ -65,28 +59,22 @@ func NewServerHTTPRequest(
 	endpoint *RouterEndpoint,
 ) *ServerHTTPRequest {
 	req := &ServerHTTPRequest{
-		gateway:     endpoint.gateway,
 		httpRequest: r,
 		queryValues: nil,
-
-		Logger: endpoint.gateway.Logger,
-		Scope:  endpoint.gateway.AllHostScope,
-
-		URL:     r.URL,
-		Method:  r.Method,
-		Params:  params,
-		Header:  NewServerHTTPHeader(r.Header),
-		metrics: &endpoint.metrics,
+		Logger:      endpoint.logger.With(logRequestFields(r)...),
```

Comment:
we do not want to put all of these fields in the logger as it's just too much data and/or PII. These log fields are only used for the request finished log line.

Reply:
Well, we mainly call `logger.Error()`, so I thought it would be good to get as much context as possible.